content/en/platform/corda/4.13/enterprise/node/corda-firewall-configuration-fields.md

```diff
@@ -34,7 +34,7 @@ An optional configuration field that specifies the file from which truststore ke
 Determines operating mode of the firewall. See [Firewall operating modes]({{< relref "corda-firewall-configuration-file.md#firewall-operating-modes" >}}).
 
 ## keyStorePassword
-The password to unlock the TLS keystore file (`<workspace>/<certificatesDirectory>/sslkeystore.jks`) containing the node certificate and private key. Due to limitations in the Artemis libraries, the private key password must be the same.
+The password to unlock the keystore files `<workspace>/certificates/sslkeystore.jks and `<workspace>/certificates/nodestore.jks` containing the node certificate and private key. Due to limitations in the Artemis libraries, the private key password must be the same.
```

Comment:
This configuration field shouldn't be updated as it refers to the firewall, so doesn't apply here. So can revert this.